web-extensions/chrome/sidebar.html

```diff
@@ -128,6 +128,7 @@
         background-color: #7fec34;
       }
     </style>
+    <script src="strings.js"></script>
```

Comment:
The sidebar is including strings.js but it is not used in scripts/load_bookmarks.js ? :)

Reply:
Oh indeed!